torch/onnx/symbolic_registry.py

```diff
@@ -120,11 +125,19 @@ def get_registered_op(opname, domain, version):
         warnings.warn("ONNX export failed. The ONNX domain and/or version are None.")
     global _registry
     if not is_registered_op(opname, domain, version):
+        raise_unsupported_operator_error(domain, opname, version)
+    return _registry[(domain, version)][opname]
+
+def raise_unsupported_operator_error(domain, opname, version):
```

Comment:
I suggest we replace this with a class:

```python
class UnsupportedOperatorError(RuntimeError):
```

It can construct the message in its `__init__`.

That makes the code at the call site much more idiomatic:
`raise UnsupportedOperatorError(...)`